### PR TITLE
Portfolio contact form

### PR DESCRIPTION
**Pull request by ridabensalem**
### Portfolio contact form
in this version we : 

- add a contact form
- add styles to the contact form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -337,11 +337,28 @@ <h1 class="skills-2">
...
+    <!-- -----------------------------------------FOOTER CONTACT FORM ----------------------------------------------- -->
+            <section class="contact-form">
+                <div class="contact-global-container">
+                 <h2 class="contact-text-container">Contact me </h2>
+                 <p class="contact-paragraph">If you have an application  you <br>are interested in developing, a <br> feature that you need built or a <br> project that needs coding.
+                    I’d <br> love to help with it</p>
+                    <form action="https://formspree.io/f/xzbwnnow" class="submit-form" method="post">
+                     <input type="text" name="name" class="username" placeholder="Yeremias" minlength="30" required="required"> 
+                    <input type="email" name="email" name="replyto" class="email" placeholder="natajayanj14@gmail.com" required="required">
+                     <textarea name="text-field" class="text-area" cols="30" rows="10" minlength="500" placeholder="Write your message here" required="required"></textarea>
```
Comment (leonardodiasb) [inline index.html]: - [ ] Your form works but the name and message inputs should have a **maximum** of 30 and 500 characters, not a **minimum**. Kindly fix this issue to meet the requirements. 👍

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @MICHAELKITH, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Good UI design. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

 - [OPTIONAL] It would be nice to add the background images to match the Figma template design.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ridabensalem) [reply index.html]: Thank you I have changed them

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @MICHAELKITH @ridabensalem, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 


## Optional suggestions

- [ ] **[OPTIONAL]:** I'm sorry that the previous reviewer didn't point this out, but this was a pair programming activity, which means that both of you needed to collaborate, but I only see commits from @ridabensalem. Please take this in mind for future projects! 🤓 🛠️ 

image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**